### PR TITLE
add trailing node

### PR DESCRIPTION
**Pull request by ericzakariasson**
**Changes**
Allows you to click at the bottom of the editor to continue editing

**Commits**
- feat: add trailing node extension
- refactor: use tailwind `@apply`
- chore: add hover state to command list button

**Issues**
Closes https://github.com/denotedxyz/denoted-issues/issues/13

### CODE REVIEW

File: apps/web/src/lib/tiptap/extensions/trailing-node.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import { Extension } from "@tiptap/core";
+import { Node, NodeType } from "@tiptap/pm/model";
+import { Plugin, PluginKey } from "prosemirror-state";
+
+// based on https://github.com/ueberdosis/tiptap/blob/40a9404c94c7fef7900610c195536384781ae101/demos/src/Experiments/TrailingNode/Vue/trailing-node.ts
```
Comment (Kenny-Gin1) [inline apps/web/src/lib/tiptap/extensions/trailing-node.ts:5]: Should these comments stay?

**Review by Kenny-Gin1 — APPROVED** [review]
LGTM!

File: apps/web/src/styles/globals.css [hunk 2]
```diff
@@ -81,18 +81,15 @@
 }
 
 .ProseMirror p.is-empty::before {
-  color: #adb5bd;
+  @apply text-slate-400 float-left h-0 pointer-events-none;
```
[the remark this answers is not in the document]
Comment (ericzakariasson) [reply apps/web/src/styles/globals.css:84]: This is not part of the tailwind config. We only add the `prose` class to the tiptap editor which is configurabe with the `typeography` plugin. What I'm modifying here is native prosemirror!

File: apps/web/src/lib/tiptap/extensions/trailing-node.ts (hunk 1 above)
Comment (ericzakariasson) [reply apps/web/src/lib/tiptap/extensions/trailing-node.ts:5]: I think it can be nice for future reference!

**Pull request merged**